packages/nextjs/src/app-beta/ClerkProvider.tsx

```diff
@@ -1,9 +1,26 @@
+import type { IsomorphicClerkOptions } from '@clerk/clerk-react/dist/types';
+import type { InitialState, PublishableKeyOrFrontendApi } from '@clerk/types';
 import React from 'react';
 
 import { initialState } from './auth';
 import { ClerkProvider as ClerkProviderClient } from './client/ClerkProvider';
 
-export function ClerkProvider({ children }: { children: React.ReactNode }) {
-  const state = initialState()?.__clerk_ssr_state || { sessionId: null, orgId: null, userId: null };
-  return <ClerkProviderClient initialState={state}>{children}</ClerkProviderClient>;
+type NextAppClerkProviderProps = {
+  children: React.ReactNode;
+  initialState?: InitialState;
```

Comment:
❓ Why do we include `initialState` here but not in `NextClerkProviderProps`

Reply:
This might be unnecessary, I'll take a look and get back to you

Reply:
Yeap, it shouldn't be there. Good catch!